formulas/wick-base/functions/wick-make-user

```diff
@@ -127,9 +136,14 @@ wickMakeUser() {
             args[${#args[@]}]=$setShell
         fi
 
-        if [[ -n "$setGroups" ]]; then
+	if [[ -n "$groups" ]]; then
+	    wickStringSplit groupsAsArray "$groups"
```

Comment:
This is redundant code. Are you able to reduce this to only one instance instead? I would suggest putting the check to see if `$groups` is set just before the detection if the user is on the system. That area could also create groups. Then here you would keep appending "-G" and `$groups` to the `$args` array.